src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Build/ImplicitlyTriggeredBuildManager.GlobalPropertiesStore.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Build
+{
+    internal sealed partial class ImplicitlyTriggeredBuildManager
```

Comment:
Most of the code being added to ImplicitlyTriggeredBuildManager is copied over from TestWindow with minor tweaks. TestWindow copy will be deleted later on.